react-app/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM node:16.15.0 as builder
+
+WORKDIR /usr/src/graphql-schema
+COPY ./graphql-schema ./
+
+WORKDIR /usr/src/app
```

Comment:
move yarn step before graphql-schema step for caching?

probably gql will be changed more often then packages list

Reply:
@chihimng  moved